extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -8,7 +8,31 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { STATE_ERROR } from '../constants';
+import * as episodeIcons from '../icons/episode-icons';
+import { STATE_ERROR, STATE_PLAYING } from '../constants';
+
+const TrackIcon = ( { isPlaying, isError, className } ) => {
+	let name;
+
+	if ( isError ) {
+		name = 'error';
+	} else if ( isPlaying ) {
+		name = 'playing';
+	}
+
+	const icon = episodeIcons[ name ];
+
+	if ( ! icon ) {
+		// Return empty element - we need it for layout purposes.
+		return <span className={ className } aria-hidden="true" />;
```

Comment:
Since there's no content in this element that would be announced via a screen reader, I think you can safely remove the `aria-hidden` attribute.

Reply:
This was required by a review here https://github.com/Automattic/jetpack/pull/15072#discussion_r397186162. Calling @getdave on this one.

Reply:
@getdave I don't see a semantic reason for adding this. There's no content to hide in this instance. Also, as a rule of thumb, we should only use `aria-hidden` where the content will be hidden for _all users_. The time to use it is when we will be providing an alternative content or the element will read duplicate, unnecessary content. https://www.w3.org/TR/wai-aria-1.1/#aria-hidden

Reply:
I see what you're getting at. However.

> Authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies only if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing **redundant or extraneous content.**

A span which has no content in it which is just sitting in the DOM for no reason is to me redundant. You only have it there as a placeholder for a logo which doesn't exist.

It's pretty minor though so feel free to do otherwise if you feel that's best.

Reply:
I think the confusion here is what they mean by "content." They're referring it to  it as "visibly rendered content." As in, something displayed on the screen. An empty span has no "visibly rendered content," so there's no visibly rendered content to remove.

Reply:
Removed in https://github.com/Automattic/jetpack/pull/15101/commits/1c60d16250f0df3c681ab2607ad38decb40c4200